dotcom-rendering/src/web/components/FrontSection.tsx

```diff
@@ -24,8 +24,6 @@ type Props = {
 	centralBorder?: 'partial' | 'full';
```

Comment:
I actually think that for `FrontContainer`, the central border should always be `full`.